terms.html

```diff
@@ -198,6 +198,11 @@
 <a>verifiable presentation</a> for processing. Other specifications might refer
 to this concept as a <dfn data-lt="relying parties">relying party</dfn>.
   </dd>
+  <dt><dfn data-lt="verification material">verification material</dfn></dt>
+  <dd>
+    Cryptographic information could be a public key or any other cryptographic data 
```

Comment:
Hmm, perhaps a slight tweak here to be more flexible:

```suggestion
    Information that could be a cryptographic public key or any other data 
```